Dockerfiles/midstream/fix_etc_passwd.sh

```diff
@@ -1,25 +1,33 @@
-#!/bin/bash -e
+#!/usr/bin/bash -e
 main() {
     local username="${SSH_AUTH_USER:-default}"
     local home
     home=$(mk_home "$username")
     setup_passwd "$username" "$home"
+    if [ "$#" -eq 0 ]; then
+      echo "Running inside a openshift environment \n
+            No arguments to script detected.\n
+            sleeping indefintely."
+      exec sleep inf
```

Comment:
This does not work.
I tried to emulate k8s condition with podman and it hangs forever.
It has to be removed.

```
sh$ podman run -ti --user=314159:0 -e HOME=/var/tmp --workdir=/tmp/ -e REGISTRY_IGNORE_TLS=1 -e IMAGE_TO_TEST=quay.io/cvpops/test-operator:v1.0-16 --name testing_midstream localhost/midstream_image:latest
314159
Running inside a openshift environment \n
            No arguments to script detected.\n
            sleeping indefintely.
```
sh# podman ps
CONTAINER ID  IMAGE                             COMMAND  CREATED        STATUS            PORTS   NAMES
c5b8d885c119  localhost/midstream_image:latest           5 minutes ago  Up 5 minutes ago          testing_midstream

sh# systemd-cgls | grep -A4 c5b8d885c11[9]
│ ├─libpod-c5b8d885c11905c84338a9a01486b4cc1d2598af352309db1e4f7e848c27ae4e.scope …
│ │ └─212305 sleep inf
│ └─libpod-conmon-c5b8d885c11905c84338a9a01486b4cc1d2598af352309db1e4f7e848c27ae4e.scope …
│   └─212293 /usr/bin/conmon --api-version 1 -c c5b8d885c11905c84338a9a01486b4c…
└─kubepods.slice 
  ├─kubepods-pod0b34b4b8_e72f_11eb_86d0_246e9616b11c.slice 
  │ ├─docker-292db097dc41725d084ea8a7e2c391624018fc8d333feb5c0a879a4f0bfa4815.scope …
  │ │ ├─1149365 /usr/bin/dumb-init -- /tmp/scripts/run
├─machine.slice
│ ├─libpod-c5b8d885c11905c84338a9a01486b4cc1d2598af352309db1e4f7e848c27ae4e.scope …
│ │ └─212305 sleep inf

```

```

Reply:
@lslebodn Inorder to run the container inside midstream. 
It is expected to run forever, Untill its killed. 
Inorder to run inside podman we have to override the endpoint with run_tests.py 
podman run -ti --entry-point run_tests.py --user=314159:0 -e HOME=/var/tmp --workdir=/tmp/ -e IMAGE_TO_TEST=quay.io/cvpops/test-operator:v1.0-16 --name testing_midstream localhost/midstream_image:latest

Reply:
If we cannot change the script that we should add comment at the top.

Please do not modify; current behaviour is expected by midstream

ot something similar

Reply:
Sure

Reply:
@lslebodn Updated

Reply:
thank you